instrumentation/opentelemetry-instrumentation-cassandra/src/opentelemetry/instrumentation/cassandra/__init__.py

```diff
@@ -65,7 +65,10 @@ def _traced_execute_async(func, instance, args, kwargs):
             if span.is_recording():
                 span.set_attribute(SpanAttributes.DB_NAME, instance.keyspace)
                 span.set_attribute(SpanAttributes.DB_SYSTEM, "cassandra")
-                span.set_attribute(SpanAttributes.NET_PEER_NAME, instance.cluster.contact_points)
+                span.set_attribute(
+                    SpanAttributes.NET_PEER_NAME,
+                    instance.cluster.contact_points,
+                )
```

Comment:
I ran `.tox/lint/bin/black .` after `tox -e lint` (which failed with `Could not find a version that satisfies the requirement opentelemetry-semantic-conventions==0.42b0.dev` for `opentelemetry-instrumentation-wsgi[test]`; I'm not sure how to point tox to my local version; I'm using `tox==3.27.1`) and it did some reformatting. I can try to revert this if I can get some pointers on the dependency issue.

Reply:
I manually reverted most of these for now